WeatherForecast/BL/CommentStrategy/WarmlyStrategy.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BL
+{
+    public class WarmlyStrategy : IChooseCommentStrategy
+    {
+        public string Comment()
+        {
+            return "its Warm";
```

Comment:
here you should use some const or enum value

Reply:
Constant added